admin/config-ui/class-configuration-page.php

```diff
@@ -85,12 +86,17 @@ public function show_wizard() {
 	 * @return array The API endpoint config.
 	 */
 	public function get_config() {
-		$config = array(
+		$service = new WPSEO_GSC_Service();
```

Comment:
Seems to me that we don't want to couple this class or this PHP component to the GSC classes.